### ISSUE TITLE
Images How to use

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/FreeUKRegProductIssues#999]
Priority 14 (0	3	8	3)

EricD said
I think one of us needs to do a write up which will eventually go into the CC and SC Help.
I could do with it now!

**Comment by Captainkirkdawson** [comment]
Agree 100% on need to write a document on how to use the image system

**Comment by edickens** [comment]
I will make a start on this.  But I am not happy about using this Story to record our discussion.  This comment goes to many people and until we have got our ideas agreed, I'd prefer to do it off list between Steve, EricB, Kirk, Bev and myself.  We need to create a document which can be edited.  ED

**Comment by Sherlock21** [comment]
I am inclined to agree with Eric D

A story on here is great for noting the final result, but its too public for really open discussion.

**Comment by SteveBiggs** [comment]
A google document shared between us would work.

**Comment by Captainkirkdawson** [comment]
Agree we should use a google document for the content. This story is to
remind us of the need to so do

Kirk Dawson
5220 Riverside Drive
Fairmont Hot Springs, B.C.
V0B 1L1

**Comment by edickens** [comment]
Never had any success with Google Documents.  I never seem to have permission to create them, nor know where FR keeps them, nor keep track of the changes as they tend to get lots of amendments and notes.
I have made a start, but need a skeleton set up somewhere which I can edit.  ED

**Comment by Captainkirkdawson** [comment]
https://docs.google.com/a/freeukgenealogy.org.uk/document/d/1sWZjkzw9VFVepNMkoiOw8Gn4Vy9C_6YQKq2wfXyNoBc/edit?usp=sharing

**Comment by edickens** [comment]
I have entered what I have done so far, but it is not for everyone to see because we need to agree some basics and it could cause confusion if those not involved read it.  Also, some of the links in it don't work.
The blue and red are exchanges between Steve and myself.  Steve uses the Image Server extensively so is the best to know what we need to make it easy to assign (allocate?) work to transcribers.

**Comment by SteveBiggs** [comment]
Is this intended to be help notes on using the website features that Bev has implemented or on the process of  CCs directly allocating images? If the latter,  we don't know exactly how the new system will work do we?

**Comment by Captainkirkdawson** [comment]
I had assumed they were a description of how we expected coordinators and others to use these capabilities

**Comment by SteveBiggs** [comment]
I agree, but I think we need to complete our discussions on the naming and how the various functions precisely work before we can write the help pages.

**Comment by SteveBiggs** [comment]
Draft help document on process of allocating image groups and assigning images completed on Gdrive here:
https://docs.google.com/document/d/1BR9kXW1I1BBf5BkRRIM4J6iGBUUQzdP65Hj846MAZiI/edit?usp=sharing

**Comment by PatReynolds** [comment]
Many thanks, Steve.  I have added a couple of comments.

**Comment by SteveBiggs** [comment]
Further updates required due to recent functional changes

**Comment by richpomfret** [comment]
Readyto be added to the help pages via refinery. @AlOneill can we pass the buck to you on this one?

**Comment by AlOneill** [comment]
@richpomfret Consider the buck passed — though with 25 pages of passive-voice heavy text and 31 images, it could take a while ...

**Comment by SteveBiggs** [comment]
@AlOneill just to confirm you have the final help docs, I have separated them out by role so there are a total of 6 docs all in this folder: https://drive.google.com/drive/folders/1hNMnyItFaiWbN6qyVFFZd9CPOV1HPpu1?usp=sharing
Please ignore anything in the sub-folder called 'Old'.

**Comment by AlOneill** [comment]
@SteveBiggs  Thanks. The docs mention another document — Uploading, viewing and downloading images — which is not in that folder. Is this available yet? Or have the plans changed?

**Comment by SteveBiggs** [comment]
That doc is now called "Working Directly with the image server through the app"

**Comment by richpomfret** [comment]
@AlOneill will you get a chance to have  a look at this over the next sprint?

**Comment by AlOneill** [comment]
Will do Transcriber instructions first, then for Coords. Made a (very small) start today — https://test3.freereg.org.uk/cms/information-for-transcribers/using-the-image-server-draft

**Comment by AlOneill** [comment]
@SteveBiggs If a transcriber chooses "Show Image Groups Available Under My Syndicate", is there an associated action to send a msg to their SC requesting more work? That is, a similar action to the one described for requesting work "By County"?

I cannot see this as my syndicate (Technical) has nothing allocated!

**Comment by SteveBiggs** [comment]
No there is not.

**Comment by AlOneill** [comment]
@SteveBiggs @edickens @Captainkirkdawson 
The pages are ready for review and can be seen at:
for transcribers, https://test3.freereg.org.uk/cms/information-for-transcribers/using-the-image-server-draft
for Coords, https://test3.freereg.org.uk/cms/information-for-coordinators/the-image-server-draft

Now is the time to propose a different name for the pages, for example, as well as to check the contents.

The figures on the Coords' page will need to be updated when the new colour scheme is in place: this will give the opportunity to to use screen shots at the ideal width (my fault for not explaining how to do this). In the meantime, fig. 4 shows how we can achieve good legibility across the widest range of window widths — I am thinking of those who would like to work with two browser windows, one for the instructions and one for the actual work.

The markup needed is a little complex and so is best done for all the remaining figures once everything else is finalised.

The situation is more difficult on the transcribers' page because of the longer sub-menus at the top of the page — we can't make the current figures more legible without introducing the need to scroll them horizontally (at all window widths). However, the instructions for transcribers are much simpler and may not require two windows. Again, any legibility issues can be dealt with when we update the figures for the new colour scheme.

**Comment by Captainkirkdawson** [comment]
@AlOneill writing how to notes on structuring images for help
@SteveBiggs viewing text
@SteveBiggs suggested @AlOneill proceed to incorporate full width updated images

**Comment by SteveBiggs** [comment]
@AlOneill are you also adding the information from the document on working directly with the Image Server to cover uploading images?

**Comment by AlOneill** [comment]
@SteveBiggs That document is only an outline: there is no actual info there on uploading images.

**Comment by AlOneill** [comment]
@SteveBiggs Will certainly add Uploading, if the instructions are detailed anywhere?

**Comment by SteveBiggs** [comment]
@AlOneill uploading is covered in the doc at section 5. Admittedly, it will need a bit of rewording as you've done for the other documents, but the basic details are there.

**Comment by AlOneill** [comment]
@SteveBiggs Aha — I see it now — didn't look closely enough.

Do you want the uploading part to be included in this initial release, or can it wait?

**Comment by edickens** [comment]
Yes, I would like it included.  CCs have images to upload.
But perhaps we need to be more open about the fact that the default upload is for images NOT to be open data.  CCs need to understand that long after they have given up the job it will be necessary to answer data questions and so look at the images.  It is no use holding them on their computers. And the images can be types of document other than IMG.

**Comment by AlOneill** [comment]
OK — have begun work on a page for the Uploading instructions.

@edickens You seem to be making 3 points, if I understand you correctly:
* images need to be marked as 'Open Data' if this is appropriate (will add a note); 
* looking to the future, we need **all** images transcribed to be on the IS (will add a note); 
* the list of example file types that can be uploaded already includes PDF — is there another format that needs a specific mention?

@SteveBiggs @Captainkirkdawson  Do either of you have "clean" copies of the images used in the document? The third image is (unhelpfully) identical to the third-last and some of the other annotations are messy enough to detract from the message, so I would like to re-do them. (There are 13 images used on the page, if that helps, and the third/third-last is numbered 2 in the download.)

Thanks!

**Comment by edickens** [comment]
Alison,
1. The default is NOT Open data so if at all possible the images should be made Open.  CCs should not just skip over this entry.
2. Even images that have been transcribed should be on the server.  This is so that any reports of data errors can be checked against the image.  This is more important for images that are Open Data as we may give researchers access to these in the future.
3. I think you can upload any file type, but only those on the list can be viewed online.  Any others will have to be downloaded to be viewed.
Thanks EricD

**Comment by SteveBiggs** [comment]
@AlOneill this document was created by Kirk so all the images came from him.

**Comment by suffolkroots** [comment]
I have images where condition of use is that they are destroyed after transcription. I am therefore not uploading them to the server. In these circumstances would it be OK to note that they can be viewed on FMP for any future amendment purposes?

**Comment by edickens** [comment]
Three points:-
- At present only SysAdmin can delete images.  The relevant CC needs to be given this permission.  That would let Julie delete them when they have been transcribed.
- Bulk deletion of images (Story already done) is needed.  At present it is one at a time.  Just not realistic long term.
- If we had permission to transcribe the images, I don't see why we cannot use FMP for corrections, but it needs a subscriptions.  Does FamilySearch have them?

**Comment by AlOneill** [comment]
I am concerned that many coords could baulk at the bulk renaming of images. And, unless they are dealing with just a handful of files, manual re-naming would be tedious and error-prone.

Should we provide more details of how to use the software suggested which is Win only? Bulk renaming on a Mac is available to all users, through Finder — have not tried it but could put together a few basic guidelines. Can't do Win — would need help/input from AN Other.

**Comment by edickens** [comment]
To rename I use a package called A to B by TGRMN Software.  I don't know if it is Windows only, but it could be.  It is very quick and easy, once you spend a few minutes finding your way around it, as you have to do with any software.
But the re-naming of images is not as important as having the correct name for transcription batches.  The important thing is that if someone has saved them to their computer, then they should be able to identify which Image Group to which they belong.  The image name also have to be unique within a Source, not just the Image Group.

**Comment by AlOneill** [comment]
@PatReynolds @SteveBiggs Whilst finishing the collateral editing for the Transcribers page (and logged in on live as a transcriber, Technical Syndicate), I found that the "Roadmap" action took me to a Google doc that required me to log in to Google to view it. Is this what *all* transcribers get? — it doesn't seem very friendly.

When logged in as Doc Manager, I got the same result.

**Comment by AlOneill** [comment]
Transcriber and Coord pages ready for review. Please check that I've not introduced any errors as a result of editing.

**Comment by edickens** [comment]
Hi Alison,
Where are they?  Test3?
Thanks Eric

**Comment by AlOneill** [comment]
@edickens Yes, on test3. Log in and select Help — you will see the main IS info page in the sub-menu. You can also follow the link from Coord Help to to Transcriber Help and hence their IS page, or use the link on the Coord IS main page.
(Not posting test3 page links directly as we are keeping public from test servers — makes sense to me!)

**Comment by edickens** [comment]
A small point.  I would not start with saying "a new image server" because these instructions should last for 50 years.  In fact I wouldn't use the word New anywhere.
"Please note that Groups are logical structures inferred from the name of the Image file" is not strictly correct.  This is how the image groups were created from the FR1 image set, but you can choose any name you like if you are uploading more images.
In Coordinator Help, after figure 7 say "Note that the Image Group name is not amendable.  If you wish to change it, or divide up a group, create a new Image Group and there is an action to move images to this new group(s).  The wrong Group can then be deleted."  Put a link in to Uploading Files to the Image Server?
After Figure 8, I suggest saying that the Unallocated status causes these image groups to appear in the list available for transcribers to select work.
After Figure 20 say that the Image Group is then taken off the report for transcribers to select more work.
Everything else looks great.

**Comment by AlOneill** [comment]
@edickens Thanks — very helpful.

I have it on my TODO list to remove all refs to a "new" IS after a suitable interval. I was thinking it would be new to some, even many, for a few months or so yet, but if "new" is already redundant, then I'll remove. @SteveBiggs What do you think? Is the IS no longer "new" already?

All other suggestions will be done ASAP — "is not amendable" will have to be replaced though :-D

**Comment by SteveBiggs** [comment]
The IS is no longer new to those who have moved from the old to new way of using it. For completely new users, we don't need to say it's new so I would agree with Eric - remove references to new.

**Comment by AlOneill** [comment]
@SteveBiggs @edickens TY — refs to "new" removed.

Have made other changes requested, but am wondering about the action to move images to a different Image Group: we don't have any instructions for this. Can you please provide brief instructions, or point me to what I've missed in the Google docs. TY

**Comment by edickens** [comment]
Looking at Test2 to find the part of the Help, I find this under Information for Coordinators.
![Capture.JPG](https://images.zenhubusercontent.com/5cb4c5e2cccdd751e9e85870/a09b3624-8e56-4122-b925-1d0a2ff083c1)
This is not correct.  The Coordinator can do the assignment.  Perhaps I should be looking at Test3?

**Comment by edickens** [comment]
Yes.  My mistake I should be using Test3.
Some suggestions:-
- It is said that the only Source is "Image Server".  We should encourage CCs to set up sources for images that they have on their computer, or if they are using books such as Phillimores.  It will then be possible to link transcriptions to the Source.
- The section about moving an image to a different Image Group is...
![Capture.JPG](https://images.zenhubusercontent.com/5cb4c5e2cccdd751e9e85870/f479728f-bde4-416f-9f53-96d3359cd088)
But look at image 12 and Edit and Re-assign are the actions to use.  The terminology should be consistent.
- It might be worth a section as some point to suggest to CCs that if the Image Group is very large and needs dividing up then the same routine as renaming an Image Group needs to be used.

Otherwise I think this section is very good and can go live.

**Comment by edickens** [comment]
PS.  It might be worth emphasising that images can only be moved within am image group, but they will find that out from the options given.  If they need to move the images to a different Image Source, such as to a different Register Type, they have to be downloaded; uploaded to the new Register Type/Source (taking care not to duplicate image filenames); then delete the images from the old Image Source/Group, which at present can only be done by System Administrators.

**Comment by AlOneill** [comment]
Ah, I was searching for 'move' whereas the text used 'transfer' — have changed 'transfer' to 'move (re-assign)' as that seems the more obvious word: 're-assign' is also used in another part of the process and so could be ambiguous unless the context is (or is made) clear.

Have added a para re more complex moves as per your 2nd (PS) comment.

**Comment by edickens** [comment]
That is fine.  Go for it.
Eric

**Comment by SteveBiggs** [comment]
In the Transcriber Help, before suggesting they search for available groups using the action buttons, we need a comment along the lines of:
"When your SC has received the email and accepted your transcription, they should assign you your next image group. If you do not receive a new assignment" and need more work, go to ......

Also, there is a redundant word "to" 
![image](https://user-images.githubusercontent.com/25303607/64706137-f1edd280-d4a8-11e9-88c5-5488983131c9.png)

**Comment by AlOneill** [comment]
@SteveBiggs TY. Changes made.
